src/agent/src/config.rs

```diff
@@ -38,6 +38,8 @@ const SECURE_STORAGE_INTEGRITY_OPTION: &str = "agent.secure_storage_integrity";
 // to solve the problem of not being able to access the specified image in some cases.
 const HTTPS_PROXY: &str = "agent.https_proxy";
 const NO_PROXY: &str = "agent.no_proxy";
+#[cfg(feature = "agent-policy")]
+const POLICY_FILE_OPTION: &str = "agent.policy_file";
```

Comment:
Does this change also allow changing the policy file path using:
1. The kernel command line, and/or
2. A pod annotation?

I vote for blocking these two means of configuration because:
a. They are relatively risky for a CoCo TEE, and 
b. We don't have (yet?) use cases that require them.

Reply:
Yes @danmihai1 it also allows from kernel cmdline ( and setting cmdline from pod annotations).
Makes sense I will remove this.
As the current use case is only for testing, the environment variable path is sufficient.